### ISSUE TITLE
The behaviour of retain selection on reload differs depending on style

### ISSUE THREAD

**Issue by gh0stwizard** [DataTables/Select#34]
Hi,
I have found that retain selection on reload is broken when `style: 'multi'`. The example can be found [here](https://datatables.net/extensions/select/examples/initialisation/reload.html). The code (shortly):
```javascript
var tableid = 'table#mytable';
...
var table = $(tableid).DataTable({
    rowId: 'Id',
    select: {
      style: 'multi'
    }
});
...
// somewhere calling DataTables by tableid
$('button').on('click', function () {
    $(tableid).DataTable().ajax.reload();
});
```
In the case above I see the table is reloading data, right after a (half) second all selected items are reset (deselected). If I use `style: 'os'` or `style: 'single'`, then Select is working as intended.

* I am using ajax for getting the data without server processing.
* DataTables 1.10.15
* Select for DataTables 1.2.2

Thanks!

**Comment by gh0stwizard** [comment]
Solved by updating to Select 1.2.6 (Select 1.2.2 is shipped with nuget version of DataTables 1.10.15, latest atm). Sorry for any inconveniences.

**Comment by DataTables** [comment]
No worries.

We don't currently publish Nugets I'm afraid - they are provided by a third party, so you'd need to contact them about updating them.
